core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -993,6 +999,18 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleApiVersionRequest(request: RequestChannel.Request) {
```

Comment:
Comment that we are not checking authorization intentionally? otherwise it will look a lot like a bug...